### PR TITLE
[sentry-native] Upgrade to 0.4.14

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

File: ports/sentry-native/portfile.cmake [hunk 1]
```diff
@@ -26,13 +26,13 @@ if (NOT DEFINED SENTRY_BACKEND)
     endif()
 endif()
 
-if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
-    set(VCPKG_CXX_FLAGS "/D_CRT_DECLARE_NONSTDC_NAMES ${VCPKG_CXX_FLAGS}")
-    set(VCPKG_C_FLAGS "/D_CRT_DECLARE_NONSTDC_NAMES ${VCPKG_C_FLAGS}")
-endif()
-
+if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
```
Comment (JonLiu1993) [inline ports/sentry-native/portfile.cmake:29]: @AenBleidd ,Thanks for your pr,I was confused, I noticed a few places that showed changes, but when I looked closely, nothing seemed to have changed, did you change anything here?

File: ports/sentry-native/portfile.cmake [hunk 2]
```diff
@@ -26,13 +26,13 @@ if (NOT DEFINED SENTRY_BACKEND)
...
+    set(VCPKG_CXX_FLAGS "/D_CRT_DECLARE_NONSTDC_NAMES ${VCPKG_CXX_FLAGS}")
+    set(VCPKG_C_FLAGS "/D_CRT_DECLARE_NONSTDC_NAMES ${VCPKG_C_FLAGS}")
+endif()
+
```
Comment (JonLiu1993) [inline ports/sentry-native/portfile.cmake:33]: @AenBleidd ,Thanks for your pr,I was confused, I noticed a few places that showed changes, but when I looked closely, nothing seemed to have changed, did you change anything here?

File: ports/sentry-native/portfile.cmake [hunk 3]
```diff
@@ -26,13 +26,13 @@ if (NOT DEFINED SENTRY_BACKEND)
...
 vcpkg_cmake_configure(
-    SOURCE_PATH "${SOURCE_PATH}"
+    SOURCE_PATH "${SOURCE_PATH}"
```
Comment (JonLiu1993) [inline ports/sentry-native/portfile.cmake:35]: Same as above

File: ports/sentry-native/portfile.cmake (hunk 2 above)
Comment (AenBleidd) [reply ports/sentry-native/portfile.cmake:33]: I made this PR with VSCode on WIndows, and either VSCode or git made automatic line ending change from `0x0A` to `0x0A 0xAD`.
I can try to remove these changes if necessary, but assuming that before everything was fine, these 'incorrect' changes were introduced by 48b4cb1cce3cb60150bbb25deec078ddd1b163c3 and now I just unified them again.

File: ports/sentry-native/portfile.cmake (hunk 3 above)
Comment (AenBleidd) [reply ports/sentry-native/portfile.cmake:35]: Same as above :)

**Review by JonLiu1993 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Pull request merged**